### ISSUE TITLE
Some questions on resnet performance

### ISSUE THREAD

**Issue by mjc92** [kuangliu/pytorch-cifar#2]
Hi,
I downloaded your codes and tested variations of resnet using the same settings,
but I can't seem to get anywhere near the accuracy mentioned (regardless of the models, all of my tests seem to plateau at around 88~89%). Are there any additional measures that you applied such as weight decay or lowering learning rates? 
Neat implementation, by the way.

**Comment by kuangliu** [comment]
You need to manually change the `lr`. 
Stop the training when loss stop decreasing, and `python main.py --resume --lr=1e-4` to continue the training.
